oc_cdtapi/ForemanAPI.py

```
@@ -699,7 +700,8 @@ def add_puppet_class_to_host_v2(self, hostname, params):
         """
         logging.debug('Reached add_puppet_class_to_host_v2')
         logging.debug('Params to be sent: %s' % params)
-        response = self.post(posixpath.join('foreman_puppet', 'api', 'hosts', hostname, 'puppetclass_ids'), headers=self.headers, data=params)
+        response = self.post(posixpath.join('foreman_puppet', 'api', 'hosts', hostname, 'puppetclass_ids'),
```

Ruff (0.14.5)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)